.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: Build
+on:
+  - push
+  - pull_request
+  - workflow_dispatch
+
+jobs:
+  build-python:
+    uses: goes-funky/workflows/.github/workflows/build-python.yaml@make-ssh-private-key-optional-python
+    with:
+      python-version: "3.9"
+      skip-mypy: true
+      skip-tests: false
+      skip-isort: true
+      skip-format: true
+      skip-lint: true
+    secrets:
+      ssh-private-key: ${{ secrets.GH_REPO_READ_SSH_KEY }}
```

Comment:
Please remove this variable entirely. There is no point in providing it if it doesn't receive a value.